src/main/kotlin/no/nav/familie/tilbake/config/IntegrasjonerConfig.kt

```diff
@@ -14,6 +14,6 @@ class IntegrasjonerConfig(@Value("\${FAMILIE_INTEGRASJONER_URL}") val integrasjo
         const val PATH_ORGANISASJON = "api/organisasjon"
         const val PATH_ARKIVER = "api/arkiv/v3"
         const val PATH_DISTRIBUER = "api/dist/v1"
-        const val PATH_OPPGAVE = "/oppgave/"
+        const val PATH_OPPGAVE = "/oppgave"
```

Comment:
Hvis du hadde brukt Uricomponetsbuilder sånn som de andre uriene er bygget med, så hadde du nok sluppet denne buggen. Den legger på slashene for deg, slik at du slipper å tenke på det.